WordPress/src/main/java/org/wordpress/android/ui/posts/PostEditorAnalyticsSession.java

```diff
@@ -113,6 +115,10 @@ public void setOutcome(Outcome newOutcome) {
         mOutcome = newOutcome;
     }
 
+    public void applyTemplate(String template) {
+        mTemplate = template;
```

Comment:
This method should also be generating a `editor_session_template_apply` event

Reply:
Thanks, updated.